sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2_and_later/async_samples/sample_send_request_async.py

```diff
@@ -0,0 +1,84 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_send_request_async.py
+
+DESCRIPTION:
+    This sample demonstrates how to make custom HTTP requests through a client pipeline.
+
+USAGE:
+    python sample_send_request_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_FORM_RECOGNIZER_ENDPOINT - the endpoint to your Form Recognizer resource.
+    2) AZURE_FORM_RECOGNIZER_KEY - your Form Recognizer API key
+"""
+
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.core.rest import HttpRequest
+from azure.ai.formrecognizer.aio import DocumentModelAdministrationClient
+
+
+async def sample_send_request():
+    endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
+    key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
+
+    client = DocumentModelAdministrationClient(
```

Comment:
Nit: it might be better to have a sample showcasing an analysis request since those are more common, but feel free to take or leave that suggestion.